Mainnet/DividendToken/DividendToken/DividendToken.cs

```diff
@@ -0,0 +1,288 @@
+﻿using Stratis.SmartContracts;
+using Stratis.SmartContracts.Standards;
+
+[Deploy]
+public class DividendToken : SmartContract, IStandardToken
+{
+
+    public ulong Dividends
+    {
+        get => PersistentState.GetUInt64(nameof(this.Dividends));
+        private set => PersistentState.SetUInt64(nameof(this.Dividends), value);
+    }
+
+    private Account GetAccount(Address address) => PersistentState.GetStruct<Account>($"Account:{address}");
```

Comment:
Should this be private for any particular reason?

Reply:
Account data is actually internal purposes only and properties of Account is not in final state. They should to be computed. The needed methods already exposed on contract as `GetDividends` and `GetTotalDividends` . Client can also calculate Withdrawn dividiends by computing value of `GetTotalDividend - GetDividends `